templates/cicd/buildspec.yml

```diff
@@ -35,7 +35,7 @@ phases:
       - envs=$(./copilot-linux env ls --json | jq '.environments[].name' | sed 's/"//g')
       # Generate the cloudformation templates.
       # The tag is the build ID but we replaced the colon ':' with a dash '-'.
-      - tag=$(sed 's/:/-/g' <<<"$CODEBUILD_BUILD_ID")
+      - tag=$(sed 's/:/-/g' <<<"$CODEBUILD_BUILD_ID" | rev | cut -c 1-128 | rev)
```

Comment:
lol can you update the comment above to also say that Docker tags need to be limited to 128 chars.

Reply:
Done. Seems silly that there doesn't seem to be a better way to truncate from the front, right?

Reply:
Just for fun: I think `sed 's/^.*\(.\{128\}\)$/\1/p'` would also do the truncation (`rev| cut| rev` definitely looks nicer)